src/www/widgets/widgets/ipsec.widget.php

```diff
@@ -68,7 +68,19 @@
         }
     }
 }
-
+// Initialize variable aggregated_data and loop through the ipsec_leases array to fetch the data for user, address and online status. Used later for div ipsec-mobile.
+$aggregated_data = array();
```

Comment:
`$aggregated_data = [];` is a bit shorter, the `array()` construction is used less often in our newer code.